src/Catalog.API/AIOptions.cs

```diff
@@ -8,11 +8,10 @@ public class AIOptions
 
 public class OpenAIOptions
 {
-    /// <summary>OpenAI API key for accessing embedding LLM.</summary>
-    public string ApiKey { get; set; }
-
-    /// <summary>Optional endpoint for which OpenAI API to access.</summary>
-    public string Endpoint { get; set; }
+    /// <summary>
+    /// Gets or sets whether OpenAI services should be enabled.
+    /// </summary>
+    public bool Enable { get; set; } = true;
```

Comment:
Is this necessary? Can whether this is enabled be based on whether an OpenAIClient is available from DI, which would in turn be based on whether it was appropriately configured?

Reply:
We used the ApiKey to detect if the service should be registered. This property is not set anymore since the Aspire component reads a connection string. We can use the existence of the connection string instead probably, and remove the `Enable` property of the other service too.